src/components/_hero.scss

```diff
@@ -1,7 +1,7 @@
 // Extension to the Design System hero component
 .hero {
   // sass-lint:disable no-duplicate-properties
-  margin-bottom: rem($spacing-large);
+  margin-bottom: rem($nds-spacing-large);
```

Comment:
Maybe we should just change https://github.com/nice-digital/nice-design-system/blob/v1/components/nds-hero/scss/hero.scss#L13 to be a large by default then there's no need to extend/ovewrite it here?